plugin/main/src/kotlinx/benchmark/gradle/BenchmarkConfiguration.kt

```diff
@@ -65,18 +63,27 @@ open class KotlinJvmBenchmarkTarget(
     val compilation: KotlinJvmCompilation
 ) : JvmBenchmarkTarget(extension, name)
 
+enum class JsBenchmarksExecutor {
+    BenchmarksJs,
+    BuiltIn
+}
+
 class JsBenchmarkTarget(
     extension: BenchmarksExtension,
     name: String,
     val compilation: KotlinJsCompilation
 ) : BenchmarkTarget(extension, name) {
-
+    var jsBenchmarksExecutor: JsBenchmarksExecutor = JsBenchmarksExecutor.BenchmarksJs
```

Comment:
This could be configured selecting built-in benchmark in gradle configuration:
```
register("js") {
    jsBenchmarksExecutor = JsBenchmarksExecutor.BuiltIn
}
```

Reply:
Now I see. Thanks.